source/documentation/using_the_api/register_components.md

```diff
@@ -1,18 +1,36 @@
-## The relationship between entries, items and records
+## The components of a register
 
-Registers are made up of:
+Registers are primarily composed of:
 
 * entries
 * items
 * records
 
+### Entries
+
 Entries contain metadata about what and when data has changed. 
 
-Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
```

Comment:
Since the word "field" has a very specific meaning in the world of registers, I'm wondering whether we should we use the word "property" here? For example, these have nothing to do with the field register as fields in an item do. Thoughts @arnau?

Reply:
Note that if we change this there's a few other places to change "field" to "property", unless we are actually talking about a field in the item.

Reply:
Sounds good to me, as you say, as long as we keep consistency across the docs.

Reply:
Would it make more sense to open a separate PR to cover all such instances/changes, in that case?

Reply:
I think it's better to do it in a separate PR so you can focus on that specifically.

Reply:
Cool, are you happy to review the PR as it stands then?